src/rust/runtime/network/ephemeral.rs

```diff
@@ -12,11 +17,10 @@ use ::rand::prelude::{
 // Constants
 //======================================================================================================================
 
+/// First private port.
```

Comment:
It would be good to comment why these values are chosen for these consts.

Reply:
These come from the RFC itself, sure I can link the RFC here.